documentation/modules/con-kafka-connect-multiple-instances.adoc

```diff
@@ -5,15 +5,22 @@
 [id='con-kafka-connect-multiple-instances-{context}']
 = Kafka Connect configuration for multiple instances
 
-If you are running multiple instances of Kafka Connect, pay attention to the default configuration of the following properties:
+If you are running multiple instances of Kafka Connect, pay attention to the default configuration of the following `config` properties:
```

Comment:
Can we say somethinf like _you have to change the default configuration_? Instead of _pay attention_? It might not be completely clear that _pay attention_ means that you have to change it - at least not for me as a foreign speaker.

Reply:
Changed to _you have to..._